force-app/main/default/classes/HandlerBeforeAfterConcept.cls

```diff
@@ -0,0 +1,93 @@
+/**  @classname  			:   HandlerBeforeAfterConcept
+  *  @author 				:   Raghav
+  *  @date 					:   01-02-2023
+  *  @description  			:   This is Handler class of TriggerAllEventConcept used to check after and before concepts of trigger context variables
+  *  @modification log		:	
+  *  1. Comments added      :   01-02-2023
+*/
+public with sharing class HandlerBeforeAfterConcept {
+/**  @methodname 	    :   beforeInsert 
```

Comment:
There should always be empty line before comment.

- [ ] Add empty line before comments, it adds more readability of your code.